.github/workflows/build.yml

```diff
@@ -2,36 +2,56 @@ name: Build
 on:
   push:
     branches:
-      - master
+    - master
```

Comment:
I don't know the CI syntax, but nDPI doesn't have neither "master" nor "main" branch (but "dev"). Is that correct?

Reply:
Changed to dev.